code/game/objects/items/his_grace.dm

```diff
@@ -0,0 +1,262 @@
+//His Grace is a very special weapon granted only to traitor chaplains.
+//When awakened, He thirsts for blood and begins ticking a "bloodthirst" counter.
+//The wielder of His Grace is immune to stuns and gradually heals.
+//If the wielder fails to feed His Grace in time, He will devour them and become incredibly aggressive.
+//Leaving His Grace alone for some time will reset His thirst and put Him to sleep.
+//Using His Grace effectively requires extreme speed and care.
+
+/obj/item/his_grace
+	name = "artistic toolbox"
+	desc = "A toolbox painted bright green. Looking at it makes you feel uneasy."
+	icon = 'icons/obj/items.dmi'
+	icon_state = "his_grace"
+	item_state = "artistic_toolbox"
+	lefthand_file = 'icons/mob/inhands/equipment/toolbox_lefthand.dmi'
+	righthand_file = 'icons/mob/inhands/equipment/toolbox_righthand.dmi'
+	w_class = WEIGHT_CLASS_GIGANTIC
+	force = 12
+	attack_verb = list("robusted")
+	hitsound = 'sound/weapons/smash.ogg'
+	var/awakened = FALSE
+	var/bloodthirst = HIS_GRACE_SATIATED
+	var/prev_bloodthirst = HIS_GRACE_SATIATED
+	var/force_bonus = 0
+	var/ascended = FALSE
+	var/victims_needed = 25
+	var/ascend_bonus = 15
+
+/obj/item/his_grace/Initialize(mapload)
+	. = ..()
+	START_PROCESSING(SSprocessing, src)
+	GLOB.poi_list |= src
+	RegisterSignal(src, COMSIG_MOVABLE_POST_THROW, .proc/move_gracefully)
+
+/obj/item/his_grace/Destroy()
+	STOP_PROCESSING(SSprocessing, src)
+	for(var/mob/living/L in src)
+		L.forceMove(get_turf(src))
+	GLOB.poi_list -= src
+	return ..()
+
+/obj/item/his_grace/attack_self(mob/living/user)
+	if(!awakened)
+		INVOKE_ASYNC(src, .proc/awaken, user)
+
+/obj/item/his_grace/attack(mob/living/M, mob/user)
+	if(awakened && M.stat)
+		consume(M)
+	else
+		..()
+
+/obj/item/his_grace/CtrlClick(mob/user) //you can't pull his grace
+	return
```

Comment:
This still allows you to right click and pull that way instead.
![8eVSa75AEZ](https://user-images.githubusercontent.com/57483089/110166113-c91f7280-7deb-11eb-9572-15190931133f.gif)